computestorage/helpers.go

```diff
@@ -0,0 +1,151 @@
+package computestorage
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"github.com/Microsoft/go-winio/pkg/security"
+	"github.com/Microsoft/go-winio/vhd"
+	"github.com/pkg/errors"
+	"golang.org/x/sys/windows"
+)
+
+// SetupContainerBaseLayer is a helper to setup a containers scratch. It
+// will create and format the vhdx inside and the size is configurable with the sizeInGB
+// parameter.
+func SetupContainerBaseLayer(ctx context.Context, layerPath, vhdName string, sizeInGB uint64) error {
+	var (
+		baseVhdPath = filepath.Join(layerPath, vhdName+"-base.vhdx")
+		diffVhdPath = filepath.Join(layerPath, vhdName+".vhdx")
+		hivesPath   = filepath.Join(layerPath, "Hives")
+		layoutPath  = filepath.Join(layerPath, "Layout")
+	)
+
+	if _, err := os.Stat(hivesPath); err == nil {
+		os.RemoveAll(hivesPath)
+	}
+	if _, err := os.Stat(layoutPath); err == nil {
+		os.RemoveAll(layoutPath)
+	}
+	if _, err := os.Stat(baseVhdPath); err == nil {
+		os.RemoveAll(baseVhdPath)
+	}
+	if _, err := os.Stat(diffVhdPath); err == nil {
+		os.RemoveAll(diffVhdPath)
+	}
+
+	createParams := &vhd.CreateVirtualDiskParameters{
+		Version: 2,
+		Version2: vhd.CreateVersion2{
+			MaximumSize:      sizeInGB * 1024 * 1024 * 1024,
+			BlockSizeInBytes: 1 * 1024 * 1024,
```

Comment:
Is this just standard? Do we want to make a constant for it just to be clearer?

Reply:
It's just what's hardcoded in HCS, I'd rather just keep the same behavior here. I can add a parameter to these calls to make it configurable but they're intended to be pretty simple e.g. just give the name, path, size and huzzah

Reply:
That's fine, but I would prefer it be a const please :)